content/desktop/wsl/_index.md

```diff
@@ -24,6 +24,22 @@ Before you turn on the Docker Desktop WSL 2 feature, ensure you have:
 - Windows 10, version 21H2 or later, or Windows 11, version 21H2 or later. For more information, see [System requirements](https://docs.docker.com/desktop/install/windows-install/#system-requirements).
 - Installed the WSL 2 feature on Windows. For detailed instructions, refer to the [Microsoft documentation](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
 
+>**Tip**
+>
+> For a better experience on WSL, consider enabling the WSL `autoMemoryReclaim` feature available since WSL 1.3.10 (experimental).
+> You can do this by adding the following to the .wslconfig file and restarting WSL.
+>
+> [experimental]
+> autoMemoryReclaim=true
```

Comment:
On the preview, this looks like it's all on one line which I'm guessing is not how it is meant to be?
https://deploy-preview-18259--docsdocker.netlify.app/desktop/wsl/

Reply:
Yes should be multiple lines, not sure why the parser placed it on a single line; any ideas on how to force the newline?

Reply:
Adding the ``` looks like it fixed it!